.github/workflows/test-python-install-script.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Setup Python env
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v3
```

Comment:
What's the reason to downgrade the `setup-python` action?

Reply:
Sorry, it was a bad rebase on my end.